src/appengine/server_flask.py

```diff
@@ -70,25 +94,67 @@ def register_routes(flask_app, routes):
 logs.configure('appengine')
 config = local_config.GAEConfig()
 
+# We need to separate routes for cron to avoid redirection.
+cron_routes = [
+    ('/home-cache', home.RefreshCacheHandler),
+    ('/testcases/cache', testcase_list.CacheHandler),
+]
+
 handlers = [
+    ('/', home.Handler if _is_oss_fuzz else testcase_list.Handler),
+    ('/bots', bots.Handler),
+    ('/bots/dead', bots.DeadBotsHandler),
     ('/configuration', configuration.Handler),
     ('/add-external-user-permission', configuration.AddExternalUserPermission),
     ('/delete-external-user-permission',
      configuration.DeleteExternalUserPermission),
+    ('/crash-stats/load', crash_stats.JsonHandler),
+    ('/crash-stats', crash_stats.Handler),
     ('/corpora', corpora.Handler),
     ('/corpora/create', corpora.CreateHandler),
     ('/corpora/delete', corpora.DeleteHandler),
+    ('/docs', help_redirector.DocumentationHandler),
+    ('/download', download.Handler),
+    ('/download/<resource>', download.Handler),
```

Comment:
@oliverchang here in the webapp2 url routing there was a feature for optional param but I was not able to find any similar feature for flask. The alternative was to add two rules, with and without variable. Is there any other alternative you would suggest or should we go with this?

Reply:
Seems fine to me.